rq/cli/cli.py

```diff
@@ -499,6 +499,9 @@ def worker_pool(
     else:
         job_class = Job
 
+    if verbose or quiet:
```

Comment:
Why is `logging_level` set to `None` if `verbose` is passed?

Reply:
Line 297 : you do exactly the same. I jut copy-pasted this code from "worker" command.
As said, I'm not an expert but I can read code !

Reply:
Ok, let's merge this in then :)